obcsim/RS232.cpp

```diff
@@ -52,6 +52,24 @@ void RS_read(msp_link_t *lnk)
         Serial.println("--------------------------------------");
       }
       break;
+    case CMD_SEND_NVM_CITIROC_CONF:
+      Serial.println("CMD_SEND_NVM_CITIROC_CONF received");
+      len = 144;
+      if (fill_commanddata(len)) {
+        Serial.println("-------- Invoking SEND_NVM_CITI_CONF -----");
+        invoke_send(lnk, MSP_OP_SEND_NVM_CITI_CONF, commanddata, len, BYTES);
+        Serial.println("--------------------------------------");
```

Comment:
Please add a few more dashes here so that the column where this print ends matches the "Invoking" print above. With the current code, what will get printed is:

```
-------- Invoking SEND_NVM_CITI_CONF -----
--------------------------------------
```

whereas what we are looking for (to keep to the conventions already present in the code) is:

```
-------- Invoking SEND_NVM_CITI_CONF -----
------------------------------------------
```